tests/unit/compatibility/test_compatibility_ruff_black.py

```
@@ -141,12 +141,18 @@ def _setup_tools(monkeypatch: pytest.MonkeyPatch) -> tuple[FakeTool, FakeTool]:
     black = FakeTool(ToolName.BLACK, can_fix=True)
     tool_map = {ToolName.RUFF: ruff, ToolName.BLACK: black}
 
-    def fake_get_tools(tools: str | None, action: str) -> ToolsToRunResult:
+    def fake_get_tools(
+        tools: str | None,
```

Ruff (0.15.6)

[warning] Unused function argument: `tools`

(ARG001)